src/crimson/net/Messenger.h

```diff
@@ -77,7 +78,13 @@ class Messenger {
                                      uint32_t min_port, uint32_t max_port) = 0;
 
   /// start the messenger
-  virtual seastar::future<> start(ChainedDispatchersRef) = 0;
+  virtual seastar::future<> start(const std::list<Dispatcher*>&) = 0;
+
+  seastar::future<> start(Dispatcher& dispatcher) {
+    std::list<Dispatcher*> dispatchers;
+    dispatchers.push_back(&dispatcher);
+    return start(dispatchers);
+  }
```

Comment:
nit, i think caller can just use `msgr->start({some_dispatcher})`.

Reply:
also it might be more consistent if pass the parameter by pointer instead of by reference.

Reply:
removed in https://github.com/ceph/ceph/pull/38360/commits/a5a77b518705123f51e15c4d9e9bb83c7969288d